api-extractor.json

```diff
@@ -0,0 +1,364 @@
+/**
```

Comment:
why do we need those docs and api-extractor at all?

Reply:
🤷‍♂️ 

I'd be fine axing 🪓